Sources/TuistCore/Utils/FileHandler.swift

```diff
@@ -75,12 +83,21 @@ public protocol FileHandling: AnyObject {
     func write(_ content: String, path: AbsolutePath, atomically: Bool) throws
 
     func glob(_ path: AbsolutePath, glob: String) -> [AbsolutePath]
+    func createSymbolicLink(_ path: AbsolutePath, destination: AbsolutePath) throws
+    func linkFile(atPath: AbsolutePath, toPath: AbsolutePath) throws
     func createFolder(_ path: AbsolutePath) throws
     func delete(_ path: AbsolutePath) throws
     func isFolder(_ path: AbsolutePath) -> Bool
     func touch(_ path: AbsolutePath) throws
 }
 
+public extension FileHandling {
+    // Helper function with default argument
+    func exists(_ path: AbsolutePath, followSymlink: Bool = true) -> Bool {
```

Comment:
This was that I wanted to provide default argument, not for the protocol (so it would not work in class's body) - but will delete it since it is not used anymore.

Reply:
Deleted 👍